### DISCUSSION TITLE
Is it possible to style open app icon's background like the active app icon's background on Windows 11?

### DISCUSSION THREAD

**Discussion by ufuksarp** [ramensoftware/windhawk-mods#476] [General]
Windows 7 taskbar would show border/background on open apps' background even if they were not the active/focused app. That behaviour was simply better in my opinion for faster focus and navigation around the taskbar.

**Comment by m417z** [comment]
Are you looking for a solution for Win10 or Win11?

**Reply by ufuksarp** [reply]
Sorry, should've added that. For Windows 11.

**Reply by m417z** [reply]
You can use [Windows 11 Taskbar Styler](https://windhawk.net/mods/windows-11-taskbar-styler). It will require some experimentation, you can check out [this comment](https://github.com/ramensoftware/windows-11-taskbar-styling-guide/issues/3#issuecomment-1873448568) for an example of customizing the background color of taskbar items.

**Reply by ufuksarp** [reply]
Well, that's a little bit over my head. Should I make a feature or a mod request to implement something basic like having the style of focused app icon for unfocused ones? Maybe for "Taskbar Labels for Windows 11"? It already has some styling options.

**Reply by m417z** [reply]
Feel free to create a new issue. Maybe somebody will create a simpler mod for this. Alternatively, depending on how badly you want it, you can play with [Windows 11 Taskbar Styler](https://windhawk.net/mods/windows-11-taskbar-styler). It's a bit advanced but really not that hard, you don't need to be a programmer to use it, especially when you have examples to start from.
